### PR TITLE
[wasm][debugger] Add array inspection to the debugger.

### PR DESCRIPTION
**Pull request by thaystg**
Add array inspection to the debugger for sized and unsized array, the example is below:
image

### CODE REVIEW

File: mono/mini/debugger-engine.h [hunk 1]
```diff
@@ -207,15 +207,7 @@ typedef struct {
...
-static void
-free_objref (gpointer value)
-{
-	ObjRef *o = (ObjRef *)value;
-
-	mono_gchandle_free_internal (o->handle);
-
-	g_free (o);
-}
+void free_objref (gpointer value);
```
Comment (luhenry) [inline mono/mini/debugger-engine.h]: Please prefix with `mono_debugger_` to "document" that it's a debugger only function.

**Comment by thaystg** [comment]
@monojenkins build failed

**Comment by luhenry** [comment]
@monojenkins build Linux WebAssembly

**Comment by thaystg** [comment]
@monojenkins build Linux WebAssembly

**Comment by thaystg** [comment]
@monojenkins build Linux WebAssembly

**Pull request merged**